directory/sail/views.py

```diff
@@ -1,7 +1,44 @@
-from django.shortcuts import render_to_response
+from django.shortcuts import render_to_response, render
 from django.http import HttpResponse
 from django.http import HttpResponseRedirect
+from sail.models import AlumnusProfile
+from django.db.models import Q
+
+fields = ['roll_number', 'name',  'batch']
+
+def createQuery(request, result, field):
+	if request.POST[field] == "1":
```

Comment:
Use 4 spaces for indentation of Python files instead of a tab. You can do that in sublime by adding this to user settings.
```
{
  "tab_size": 4,
  "translate_tabs_to_spaces": true
}
```
Please amend these changes by:
```
git add .
git commit --amend
git push --force feature/multiSearch
```
Test the code before pushing and let me know when pushed.